assets/js/googlesitekit/datastore/user/feature-tours.js

```diff
@@ -231,8 +231,8 @@ const baseControls = {
 				}
 
 				// Only tours with a version after a user's initial Site Kit version should qualify.
-				const initialVersion = registry
-					.select( CORE_USER )
+				const initialVersion = await registry
+					.__experimentalResolveSelect( CORE_USER )
```

Comment:
As discussed [above](https://github.com/google/site-kit-wp/pull/6203/files#r1051087530), this is necessary to fix an issue with referring to resolver-derived values in a `useMount`. The `useMount` call that results in this control running is here:

https://github.com/google/site-kit-wp/blob/d0cd49c2d47d758ce7991fd7778a35bb755252a3/assets/js/components/FeatureTours.js#L37-L39